app/dashboard/models.py

```diff
@@ -2641,8 +2641,7 @@ def get_eth_sum(self, sum_type='collected', network='mainnet', bounties=None):
 
         """
         eth_sum = 0
-
-        if not bounties:
+        if bounties is None:
```

Comment:
if `bounties.exists()` was `0`, then this condition was hit and we'd lookup the `get_funded_bounties()` again.. thus hurting performance